lib/cocoapods/command/deintegrate.rb

```diff
@@ -1,5 +1,7 @@
 module Pod
   class Command
+    # @since 1.0.0.beta.1
```

Comment:
What if we add a custom tag `@CocoaPods` and use it like `@since`? http://www.rubydoc.info/gems/yard/file/docs/Tags.md#Adding_Custom_Tags